src/operator/cudnn_convolution-inl.h

```diff
@@ -210,6 +210,7 @@ class CuDNNConvolutionOp : public Operator {
       typename DataType<DType>::ScaleType beta = 0.0f;
       typename DataType<DType>::ScaleType beta_add = 1.0f;
       if (!param_.no_bias) {
+        CHECK_NE(req[conv::kBias], kNullOp);
```

Comment:
This line was introduced to mirror the handling seen in cudnn_deconvolution-inl.h.  To summarize the behavior as coded here for the 4 possible cases:

param_.no_bias == true, req[conv::kBias] == kNullOp        - avoids backward-to-bias kernel
param_.no_bias == true, req[conv::kBias] != kNullOp         - avoids backward-to-bias kernel
param_.no_bias == false, req[conv::kBias] == kNullOp       - throws exception
param_.no_bias == false, req[conv::kBias] != kNullOp        - runs backward-to-bias kernel

Would you prefer a different behavior?